### PR TITLE
CI Manual Tests

### PR DESCRIPTION
**Pull request by SimonRichardson**
## Description of change

The following starts to add manual machines so we can add a cloud, then
we can bootstrap to that cloud.

This only works for LXD for now, but at a later date, we should add AWS/GCE.

## QA steps

```sh
cd tests && ./main.sh manual
```

### CODE REVIEW

**Comment by SimonRichardson** [comment]
https://trello.com/c/AMDoFvCP/2427-manual-integration-tests

**Comment by manadart** [comment]
Can we have a clean-up step to remove the created profile?
It seems to leave them around when I run locally.

**Comment by SimonRichardson** [comment]
@manadart added the cleanup of profiles.

**Comment by SimonRichardson** [comment]
$$merge$$

**Comment by SimonRichardson** [comment]
$$merge$$

> 16:53:38 Error: websocket: close 1006 (abnormal closure): unexpected EOF

**Comment by SimonRichardson** [comment]
$$merge$$ 

> FAIL: worker_test.go:505: WorkerSuite.TestInvalidDeploymentChange

File: tests/suites/manual/deploy_manual.sh [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+    file="${TEST_DIR}/test-${name}.txt"
+
+    bootstrap "${cloud_name}" "test-${name}" "${file}"
+
+    juju add-machine ssh:ubuntu@"${addr_m1}" >"${TEST_DIR}/add-machine-1.log" 2>&1
+    juju add-machine ssh:ubuntu@"${addr_m2}" >"${TEST_DIR}/add-machine-2.log" 2>&1
+
+    juju enable-ha >"${TEST_DIR}/enable-ha.log" 2>&1
+
+    juju deploy percona-cluster
```
Comment (howbazaar) [inline tests/suites/manual/deploy_manual.sh]: Where does this get deployed?

Comment (SimonRichardson) [reply tests/suites/manual/deploy_manual.sh]: Unsure tbh, I'm going to revisit this next  available CI day.

Comment (SimonRichardson) [reply tests/suites/manual/deploy_manual.sh]: We want to add aws to this setup as well, so I'll generalize this when that happens

**Pull request merged**